plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -38,17 +46,21 @@ export class GithubPublisher implements PublisherBase {
     });
 
     return new GithubPublisher({
-      token: config.token,
+      token: config.token || '',
```

Comment:
only rely on the credential provider, no need to pass on the token

Reply:
I stopped passing the token from the fromConfig function and depend on the credential provider to get the token.